packages/devtools_app/lib/src/eval_on_dart_library.dart

```diff
@@ -14,6 +16,22 @@ import 'globals.dart';
 import 'inspector/inspector_service.dart';
 import 'vm_service_wrapper.dart';
 
+abstract class Disposable {
+  bool get disposed;
+
+  void dispose();
+}
+
+class IsAlive extends Disposable {
```

Comment:
I'm not seeing a need for two classes in this CL. Just make this the default implementation of Disposable and annotate dispose with mustCallSuper(). In dart someone can implement a class if they need to use it as an interface.